recipes/graphium/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "graphium" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graphium-{{ version }}.tar.gz
+  sha256: fbb337f403fbbf23e7d3eea8ea6b7934723c4d7ae056e730239bce0713f9c19c
+
+build:
+  entry_points:
+    - graphium = graphium.cli:main_cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.10
+    - click
+    - loguru
+    - omegaconf >=2.0.0
+    - tqdm
+    - numpy
+    - scipy >=1.4
+    - pandas >=1.0
+    - scikit-learn
+    - fastparquet
+    - matplotlib-base >=3.0.1
+    - seaborn
+    - fsspec >=2021.6
+    - s3fs >=2021.6
+    - gcsfs >=2021.6
+    - platformdirs
+    - pytorch >=1.10.2,<2.0
+    - tensorboard
+    - pytorch-lightning >=1.9
+    - torchmetrics >=0.7.0,<0.11
+    - hydra-core >=1.0
+    - ogb
+    - pytorch_geometric >=2.0
+    - wandb
+    - optuna
+    - mup
+    - pytorch_sparse >=0.6
+    - pytorch_cluster >=1.5
+    - pytorch_scatter >=2.0
+    - datamol >=0.10
+    - mordred
+
+test:
+  imports:
+    - graphium
+  commands:
+    - graphium --help
```

```suggestion
    - graphium --version | grep {{ version }}
    - pip check
```
(if the former is available)

---

Good idea for --version. I already tried pip check and it fails but not because of that package.